src/html/index.html

```diff
@@ -0,0 +1,188 @@
+                        </footer>
+                    </div>
+                </article>
+            </div>
+
+            <div class="article-add">
+                <div class="article-add__button">
+                    <button class="article-add__button-add" id="article-add__button">Добавить</button>
+                </div>
+                <form class="article-add__form form" onsubmit="return false;">
+                    <div class="form__inputs">
+                        <label id="article-text__name" class="article-text-name">Название статьи</label>
+                        <input type="text" id="article-text-name">
+                        <label id="article-text__url" class="article-text-url">Ссылка на картинку</label>
+                        <input type="text" id="article-text-url">
+                        <label id="article-text__description" class="article-text__description">Описание</label>
+                        <textarea id="article-text-description"></textarea>
+                        <div class="form__buttons">
+                            <button type="reset" class="article-add__button-cancel" id="article-add__button-cancel">Отмена</button>
+                            <button type="submit" class="article-add__button-create" id="article-add__button-create">Создать</button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div class="faq container" id="faq">
+                <h3 class="faq-title">FAQ</h2>
+                <dl class="faq-page">
+                    <dt class="faq-page__question">Вопрос №0: ????????</dt>
+                    <dd class="faq-page__answer">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Velit saepe sequi, illum facere
+                        necessitatibus cum aliquam id illo omnis maxime, totam soluta voluptate amet ut sit ipsum
+                        aperiam.</dd>
+        
+                    <dt class="faq-page__question">Вопрос №1: Почему интернет такой большой, а искомых решений так мало?</dt>
+                    <dd class="faq-page__answer">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Velit saepe sequi, illum facere
+                        necessitatibus cum aliquam id illo omnis maxime, totam soluta voluptate amet ut sit ipsum
+                        aperiam.</dd>
+        
+                    <dt class="faq-page__question">Вопрос №2: Вопрос Вселенной, Жизни и всего такого</dt>
+                    <dd class="faq-page__answer">42</dd>
+                </dl>
+            </div>
+        </div>
+    </div>
+
+    <footer class="page__footer"> 
+        <div class="container">
+            <p class="footer__copywrite">© 2022 Институт космических и информационных технологий СФУ</p>
+        </div>
+    </footer> 
+</body>
+<!-- <script src="https://code.iconify.design/iconify-icon/1.0.1/iconify-icon.min.js"></script> -->
+<script  rel="stylesheet" src="assets/js/main.js"></script> 
```

Comment:
rel="stylesheet" у скрипта нет и убрать закомментированные js библиотеки
```suggestion
<script src="assets/js/main.js"></script> 
```